src/Typedefs/DependantLookup.idr

```diff
@@ -0,0 +1,25 @@
+||| This module is by itself because its only function takes ages to compile
+module DependantLookup
+
+import Data.Vect
+
+||| Lookup for maps with a shared index
+public export
+depLookup : DecEq t => {w : t} -> {f, g : t -> Type} ->
+             (vs : Vect n (v : t ** (f v, g v))) -> f w -> DecEq (f w) =>
```

Comment:
Can we move `DecEq (f w)` to the previous line?

Reply:
Yes, fixed